arkane/encorr/data.py

```diff
@@ -106,6 +106,12 @@
         'H': -0.50142, 'N': -54.57367, 'O': -75.04550
     },
 
+    'm062x-gd3_jun-cc-pvtz': {
```

Comment:
I'm familiar with `d3`, not `gd3`. Just making sure the `g` here is not a typo. Would you know what it stands for?
Also, please replace the underscore with a `/`

Reply:
yes, g stands for `Grimme`.  The keyword in Gaussian 16 is `GD3`, which is why I included the `g` in there.  However, I think it is more commonly known just `D3`, so I'll remove the `g`.

Reply:
Thinking about this more...`d3` shouldn't have an effect on the atom energies.  So `m062x-d3/jun-cc-pvtz` atom energies should be the same as `m062x/jun-cc-pvtz`.  Should I remove `d3` from this model chemistry name?

Reply:
Really? I thought it should. If it doesn't then we might as well remove it, I'd love to learn more about this issue.
@dranasinghe / @cgrambow, could you also comment? Does adding dispersion really not affect the energy?

Reply:
Dispersion is an interaction between atoms, so individual atoms are not affected.

Reply:
I agree that dispersion will not effet atom energies but here we are considering atom correction such that energies will agree with the standard state. Since dispersion effect H2, O2, and molecules "correction" for the atom energy will be affected.

Reply:
I did a quick calculation for butanol m062x/jun-cc-pvtz and m062x-GD3/jun-cc-pvtz energy in Hartrees was -233.638900464 Eh and -233.639125055 Eh respectively. If you consider the same atom corrections,  the standard formation enthalpy for m062x-GD3/jun-cc-pvtz  will be wrong by ~0.58 kJ/mol.

Reply:
Thanks all for your comments!
To conclude: in this case, calculated atom corrections w/ and w/o D3 are the same. However, if we take the approach of deriving atom corrections from the standard states as @dranasinghe mentioned, then it does matter. But this is probably not our case (I think here we let BAC's correct for that, though we do have instances in Arkane where atom corrections relate to standard states).
We do, however, need to keep the `D3` in the key label here, since these corrections must correspond to the BAC's, where dispersion does matter of course.